guide/daily-spending-wallet/security.md

```diff
@@ -10,15 +10,62 @@ redirect_from:
 main_classes: -no-top-padding
 image: https://bitcoin.design/assets/images/guide/daily-spending-wallet/security/protecting-a-wallet-preview.png
 image_base: /assets/images/guide/daily-spending-wallet/security/
+images_checklists:
+    - file: security-checklist
+      modalImage: security-checklist-big
+      alt: A security checklist screen
+      caption: An example checklist for security settings.
+    - file: inadequate-security
+      modalImage: inadequate-security-big
+      alt: A security checklist screen with a note about the users security not being appropriate to their funds stored
+      caption: In clear situations, strong recommendations can be made to switch to a better security configuration.
+    - file: checklist-option-details
+      modalImage: checklist-option-details-big
+      alt: An app screen with detailed information about how cloud backups work.
+      caption: Users should be able to easily access information about how critical features work.
+images_reminders:
+    - file: security-reminder-modal
+      modalImage: security-reminder-modal-big
+      alt: Screen recommending user to enable wallet spend limits
+      caption: A full-scren modal with a specific recommendation.
+    - file: home-security-reminder
+      modalImage: home-security-reminder-big
+      alt: Wallet home screen with a reminder to review security settings
+      caption: A more subtle inline reminder that takes users to the security checklist.
+images_pin:
+    - file: choose-pin
+      modalImage: choose-pin-big
+      alt: Choosing a pin screen
+      caption: When choosing a PIN, the user is also informed about how it is used.
+    - file: enter-pin
+      modalImage: enter-pin-big
+      alt: Enter pin screen
+      caption: PIN entry could be limited to a certain amount of unsuccessful attempts per day.
+    - file: enter-pin-scrambled
+      modalImage: enter-pin-scrambled-big
+      alt: Enter pin screen with scrambled numbers
+      caption: Scrambling numbers can make it harder for bystanders to see what was typed.
```

Comment:
`enter-pin` and `enter-pin-scrambled` both appear to be the exact same image. An error?

Reply:
Different is that the keypad numbers are scrambled (not `1-2-3...` but `3-7-8...`).